src/docker/build.nim

```diff
@@ -7,6 +7,7 @@
 
 import std/json
 import ".."/[
+  chalk_common,
```

Comment:
no need to import `chalk_common`. we actually dont import it anywhere, except maybe a few places due to:

https://github.com/crashappsec/chalk/blob/73b7ef84c5da75d3b40fc00ffd7d6ee42c30c066/src/config.nim#L10-L11

which exports everything from `run_management` which in turn:

https://github.com/crashappsec/chalk/blob/73b7ef84c5da75d3b40fc00ffd7d6ee42c30c066/src/run_management.nim#L13-L14

therefore most modules in chalk simply import `config` and that automatically imports all the base stuff

Reply:
config is already imported below. lets remove

```suggestion
```